pom.xml

```diff
@@ -26,6 +26,7 @@
   </parent>
   <groupId>org.apache.spark</groupId>
   <artifactId>spark-parent_2.11</artifactId>
+  <!-- When updating version make sure to update ./python/pyspark/version.py as well -->
```

Comment:
@JoshRosen so we already update this implicitly using release-tag.sh - this is just the version for dev builds.